checkstyle/.scalafmt.conf

```diff
@@ -12,7 +12,10 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-docstrings = JavaDoc
+version = 3.5.8
+docstrings.style = Asterisk
+docstrings.wrap = false
```

Comment:
This keeps the current docstrings intact (i.e. forces scalafmt to ignore docstrings). Turning it on messes with manual html formatting that is used in docstrings